staging/src/k8s.io/client-go/util/cert/pem.go

```diff
@@ -91,6 +119,37 @@ func ParsePrivateKeyPEM(keyData []byte) (interface{}, error) {
 	return nil, fmt.Errorf("data does not contain a valid RSA or ECDSA private key")
 }
 
+// ParsePublicKeysPEM is a helper function for reading an array of rsa.PublicKey or ecdsa.PublicKey from a PEM-encoded byte array.
+// Reads public keys from both public and private key files.
```

Comment:
The comment was retained from the function it replaces here: https://github.com/kubernetes/kubernetes/pull/41611/files#diff-cad4f907f9a840865e4ccf6d2bdf7e7bL90

However, looking over the code, as written does not actually parse out public keys from private key files.. it simply discards them as they wouldn't match "PUBLIC KEY" in the pem headers.

Fixing functionality to match intended function